docs/README.md

```diff
@@ -107,6 +106,6 @@ an issue:
 * [Debug Instructions (Windows)](development/debug-instructions-windows.md)
 * [Setting Up Symbol Server in debugger](development/setting-up-symbol-server.md)
 * [Documentation Styleguide](styleguide.md)
-* [Upgrading Chrome](development/upgrading-chrome.md)
+* [Upgrading Chromium](development/upgrading-chromium.md)
```

Comment:
In order to avoid breaking links, please create an empty file that links to the new file, using the following content:

```
echo "Moved to [upgrading-chromium.md](upgrading-chromium.md)" > updgrading-chrome.md
```